packages/buidler-core/src/internal/core/errors.ts

```diff
@@ -151,17 +260,17 @@ You probably imported @nomiclabs/buidler instead of @nomiclabs/buidler/config`
   NETWORK: {
     CONFIG_NOT_FOUND: {
       number: 100,
-      message: 'Network "%s" not defined'
+      message: "Network %network% doesn't exist"
     },
     INVALID_GLOBAL_CHAIN_ID: {
       number: 101,
       message:
-        "Buidler was set to use chain id %s, but connected to a chain with id %s."
+        "Buidler was set to use chain id %configChainId%, but connected to a chain with id %connectionChainId%."
     },
-    INVALID_TX_CHAIN_ID: {
+    /* DEPRECATED */ INVALID_TX_CHAIN_ID: {
```

Comment:
Why was this deprecated?

Reply:
This can never happend. A previous version of Buidler handled locally signed txs in an incorrect way that sometimes led to this error.

Reply:
Will add some explanation.